platform.py

```diff
@@ -23,22 +23,30 @@ def is_embedded(self):
         return True
 
     def configure_default_packages(self, variables, targets):
-        if variables.get("board"):
+        board = variables.get("board")
+
+        if board:
             upload_protocol = variables.get("upload_protocol",
-                                            self.board_config(
-                                                variables.get("board")).get(
+                                            self.board_config(board).get(
                                                     "upload.protocol", ""))
             if "tool-nrfjprog" in self.packages and \
-                    upload_protocol != "nrfjprog":
+                    upload_protocol != "nrfjprog" and \
+                    "bootloader" not in targets:
                 del self.packages["tool-nrfjprog"]
 
+            if self.board_config(board).get("build.bsp.name", "nrf5") == "adafruit":
+                self.frameworks['arduino']['package'] = "framework-arduinoadafruitnordicnrf5"
+
+        if "tool-nrfjprog" in self.packages and "bootloader" in targets:
```

Comment:
The same, please remove these lines

Reply:
These are needed to auto download the Adafruit BSP and `nrfjprog` (not `nrfutil`) when needed.

Reply:
We don't have `tool-nrfjprog` package.

Reply:
Yeah you do ;-) https://github.com/platformio/platform-nordicnrf52/blob/develop/platform.json#L53:L57 works great!